app/assets/javascripts/initializers/initializeBroadcast.js

```diff
@@ -81,20 +81,27 @@ function renderBroadcast(broadcastElement, data) {
 
 /**
  * A function to determine if a broadcast should render.
- * Does not render a broadcast on the `/new` route or in an iframe.
+ * Does not render a broadcast in an iframe, or on `/connect` and `/new` routes.
  * Does not render a broadcast if the current user has opted-out,
  * if the broadcast has already been inserted, or if the key for
  * the broadcast's title exists in localStorage.
  *
- * If the broadcast exists in the DOM but was hidden by the articleForm,
- * the function will re-display it again by adding a class.
- *
  * @function initializeBroadcast
  */
 function initializeBroadcast() {
+  const shouldHideBroadcast = window.location.pathname.match(
+    /^(?:\/connect|\/new)/,
```

Comment:
oh what's that oh it's just vaidehi beefing up her regex chops 💪 

the nice thing about this is that we should always be able to check this regex to see which pathnames we do _not_ want to render an announcement broadcast on! at some point maybe we want to extract this out into a constant, but since it's small, i figured it would be fine to leave it as is for now.